<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DvdClip</class>
 <widget class="QDialog" name="DvdClip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DvdClip</string>
  </property>
  <layout class="QVBoxLayout" name="theMainLayout" stretch="0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="theClipTopLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="theClipTop"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="theDisplayLayout" stretch="0,1,0">
     <item>
      <widget class="QSpinBox" name="theClipLeft"/>
     </item>
     <item>
      <widget class="QLabel" name="theDisplay">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="theClipRight"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="theClipBottomLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="theClipBottom"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="theButtonLayout" stretch="1,0,0">
     <item>
      <layout class="QVBoxLayout" name="theFrameLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="thePrevFrameButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="theFrameLabel">
           <property name="text">
            <string>Time: 0 seonds</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="theConfidenceLabel">
           <property name="text">
            <string>Confidence: 0%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="theNextFrameButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="theFrameSlider">
         <property name="pageStep">
          <number>120</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="theAnalyzeButton">
       <property name="text">
        <string>&amp;Analyze</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="theButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="thePrevFrameAction">
   <property name="text">
    <string>Previous Frame</string>
   </property>
  </action>
  <action name="theNextFrameAction">
   <property name="text">
    <string>Next Frame</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>theButtons</sender>
   <signal>accepted()</signal>
   <receiver>DvdClip</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>theButtons</sender>
   <signal>rejected()</signal>
   <receiver>DvdClip</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
